### PR TITLE
Refactoring network log

### PR DESCRIPTION
**Pull request by pillip8282**

### CODE REVIEW

File: external/netutils/netlib_setmacaddr.c [hunk 1]
```diff
@@ -1,7 +1,7 @@
 /****************************************************************************
  *
  * Copyright 2016 Samsung Electronics All Rights Reserved.
- *
+nn *
```
Comment (jeongchanKim) [inline external/netutils/netlib_setmacaddr.c:4]: Please revert unnecessary change :)

File: framework/src/wifi_manager/wifi_manager_message.c [hunk 2]
```diff
@@ -62,7 +61,11 @@ static inline int _recv_message(int fd, void *buf, int buflen)
 	while (1) {
 		int res = read(fd, buf + received, buflen - received);
 		if (res < 0) {
-			MESSAGE_ERROR;
+			int err_no = get_errno();
+			if (err_no == EAGAIN || err_no == EINTR) {
+				continue;
+			}
+			NET_LOGE(TAG, "read error %d\n", errno);
```
Comment (jeongchanKim) [inline framework/src/wifi_manager/wifi_manager_message.c:68]: I think we can use `err_no` instead of calling `errno`.

File: framework/src/wifi_manager/wifi_manager_state_table.h [hunk 3]
```diff
@@ -0,0 +1,9 @@
+/* handler, str*/
```
Comment (jeongchanKim) [inline framework/src/wifi_manager/wifi_manager_state_table.h:1]: Is it okay not to add Samsung license?

File: framework/src/wifi_manager/wifi_manager_event_table.h [hunk 4]
```diff
@@ -0,0 +1,17 @@
+/*  str, type, description */
```
Comment (jeongchanKim) [inline framework/src/wifi_manager/wifi_manager_event_table.h:1]: Please add Samsung License.

File: framework/src/wifi_manager/wifi_manager_dhcpc.c [hunk 5]
```diff
@@ -22,11 +22,14 @@
 #include <string.h>
 #include <sys/types.h>
 #include <netutils/netlib.h>
+#include <tinyara/net/netlog.h>
 #include <wifi_manager/wifi_manager.h>
 #include "wifi_manager_utils.h"
 #include "wifi_manager_error.h"
 #include "wifi_manager_dhcp.h"
 
+#define TAG "[WM]"
```
Comment (j11591) [inline framework/src/wifi_manager/wifi_manager_dhcpc.c:31]: How about change [DHCPC]?

File: framework/src/wifi_manager/wifi_manager_dhcps.c [hunk 6]
```diff
@@ -20,12 +20,12 @@
...
 #include <stdint.h>
 #include <string.h>
-
 #include <sys/types.h>
 #include <netutils/netlib.h>
+#include <tinyara/net/netlog.h>
 #include "wifi_manager_error.h"
 #include "wifi_manager_dhcp.h"
-#include "wifi_manager_log.h"
+#define TAG "[WM]"
```
Comment (j11591) [inline framework/src/wifi_manager/wifi_manager_dhcps.c:28]: How about changing [DHCPS]?

File: framework/src/wifi_manager/wifi_manager_dhcpc.c (hunk 5 above)
Comment (pillip8282) [reply framework/src/wifi_manager/wifi_manager_dhcpc.c:31]: it's wifi_manager module so I think wm is better.

File: framework/src/wifi_manager/wifi_manager_dhcps.c (hunk 6 above)
Comment (pillip8282) [reply framework/src/wifi_manager/wifi_manager_dhcps.c:28]: it's wifi_manager module so I think wm is better.

**Review by sangwon03 — APPROVED** [review]
LGTM
